# Generated by Django 3.1.4 on 2021-12-13 08:14

from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('foto_de_perfil', '0002_auto_20211109_2114'),
    ]

    operations = [
        migrations.AddField(
            model_name='foto',
            name='f_braco_forca',
            field=models.ManyToManyField(blank=True, related_name='f_braco_forca', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='foto',
            name='f_cara_coracao',
            field=models.ManyToManyField(blank=True, related_name='f_cara_coracao', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='foto',
            name='f_cara_dinheiro',
            field=models.ManyToManyField(blank=True, related_name='f_cara_dinheiro', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='foto',
            name='f_cara_surridente',
            field=models.ManyToManyField(blank=True, related_name='f_cara_surridente', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='foto',
            name='f_cara_triste',
            field=models.ManyToManyField(blank=True, related_name='f_cara_triste', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='foto',
            name='f_nao_gosto',
            field=models.ManyToManyField(blank=True, related_name='f_nao_gosto', to=settings.AUTH_USER_MODEL),
        ),
    ]
